starfish/test/dataset_fixtures.py

```diff
@@ -208,14 +208,54 @@ def synthetic_single_spot_3d():
 
 @pytest.fixture()
 def synthetic_two_spot_3d():
-    from scipy.ndimage.filters import gaussian_filter
     data = np.zeros((10, 100, 100), dtype=np.uint16)
     data[4, 10, 90] = 1000
     data[6, 90, 10] = 1000
     data = gaussian_filter(data, sigma=2)
     return data
 
 
+def synthetic_two_spot_3d_2round_2ch() -> ImageStack:
```

Comment:
This provides a very general multi-spot ImageStack to test all the spot-finding methods.